webpack/assets/javascripts/react_app/routes/common/reducerHOC/reducerHOC.fixtures.js

```diff
@@ -0,0 +1,13 @@
+import { combineReducers } from 'redux';
+import withDataReducer from './withDataReducer';
+import withQueryReducer from './withQueryReducer';
+
+export const reducer = combineReducers({
+  data: withDataReducer('TEST'),
+  query: withQueryReducer('TEST'),
+});
```

Comment:
Shouldn't we test each HOC separately? that way e.g. change to one doesn't require changing all snapshots. I don't think we need to test that `combineReducers` works properly - that's redux's job.